File: FeedbackSystem/Models/Feedback.cs
namespace FeedbackSystem.Models
{
    public class Feedback
    {
        public int FeedbackId { get; set; }
        public string Content { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
        public bool IsApproved { get; set; } = false;
        public int UserId { get; set; }
        public User User { get; set; }
        public int CategoryId { get; set; }
        public Category Category { get; set; }
    }
}

File: feedback-system-client/src/app/models/feedback.ts
// Student ID: 00016119

import { User } from "./user";
import { Category } from "./category";

export interface Feedback {
    feedbackId: number;
    content: string;
    createdAt: string;
    isApproved: boolean;
    userId: number;
    categoryId: number;
    user?: User;
    category?: Category;
  }
  